### DISCUSSION TITLE
mspider/user-guide/service-mesh/integrate-mesh/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1190] [Announcements]
# mspider/user-guide/service-mesh/integrate-mesh/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-mesh/integrate-mesh/

**Comment by liyuerich** [comment]
选择外接网格，填写网格配置信息

截图中的信息能否为真实的和网格相关的信息？
